tests/test_compile_error.rs

```diff
@@ -5,6 +5,8 @@ fn test_compile_errors() {
     t.compile_fail("tests/ui/invalid_macro_args.rs");
     t.compile_fail("tests/ui/invalid_property_args.rs");
     t.compile_fail("tests/ui/invalid_pyclass_args.rs");
+    t.compile_fail("tests/ui/invalid_pymethod_names.rs");
```

Comment:
This test already existed but seemed to accidentally be lost from the compile error test list.

Reply:
Ooops, I accidently removed it when tackling the CI problem. Thanks!